DatabaseTemplate/Modules/create-sk-template.psm1

```diff
@@ -0,0 +1,137 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+
+$ErrorActionPreference = "Stop"
+
+& "$PSScriptRoot\..\..\logistics\scripts\modules\load-path-resolver.ps1"
```

Comment:
if you move this and the test harness config into the other repo you should be able to set the path like this:
```suggestion
& "$PSScriptRoot\..\..\Ed-Fi-ODS-Implementation\logistics\scripts\modules\load-path-resolver.ps1"
```

Reply:
updated